include/esp_can_update.h

```diff
@@ -44,37 +45,35 @@ class CANUpdate
     enum class MessageType : uint8_t
     {
         kUpdateStart = 0,
-        kUpdateData = 1,
-        kMd5 = 2
+        kMd5 = 1
     };
     MakeUnsignedCANSignal(MessageType, 0, 8, 1, 0) message_type_{};
     MakeUnsignedCANSignal(uint32_t, 8, 32, 1, 0) update_length_{};
     MakeUnsignedCANSignal(uint16_t, 8, 8, 1, 0) update_md5_idx_{};
     MakeUnsignedCANSignal(uint32_t, 16, 32, 1, 0) update_md5_{};
-    MakeUnsignedCANSignal(uint32_t, 8, 24, 1, 0) data_block_index_{};
-    MakeUnsignedCANSignal(uint32_t, 32, 32, 1, 0) update_data_{};
 
     MultiplexedSignalGroup<1, MessageType> length_signal_group_{MessageType::kUpdateStart, update_length_};
-    MultiplexedSignalGroup<2, MessageType> data_signal_group_{
-        MessageType::kUpdateData, data_block_index_, update_data_};
     MultiplexedSignalGroup<2, MessageType> md5_signal_group_{MessageType::kMd5, update_md5_idx_, update_md5_};
 
+    MakeUnsignedCANSignal(uint8_t, 0, 8, 1, 0) data_block_index_low_{};
+    MakeUnsignedCANSignal(uint64_t, 8, 56, 1, 0) update_data_{};
+
     MakeUnsignedCANSignal(uint32_t, 0, 24, 1, 0) update_block_idx_{};
     MakeUnsignedCANSignal(bool, 24, 1, 1, 0) received_len_{};
     MakeUnsignedCANSignal(bool, 25, 1, 1, 0) received_md5_{};
     MakeUnsignedCANSignal(bool, 26, 1, 1, 0) written_{};
 
     CANTXMessage<4> update_progress_message_{
-        can_interface_, kUpdateId + 1, 4, 10, timer_group_, update_block_idx_, received_len_, received_md5_, written_};
+        can_interface_, kUpdateId + 2, 4, 10, timer_group_, update_block_idx_, received_len_, received_md5_, written_};
```

Comment:
Why +2?  Can the 2 be given a descriptive constant name?   (more below)

Reply:
That's just the ID offset of that message, could be a const but this is the only place it's used and I don't think it's necessary